x-pack/plugins/task_manager/server/task_store.ts

```diff
@@ -294,6 +294,22 @@ export class TaskStore {
     }
   }
 
+  /**
+   * Bulk removes the specified tasks from the index.
+   *
+   * @param {SavedObjectsBulkDeleteObject[]} savedObjectsToDelete
+   * @returns {Promise<void>}
+   */
+  public async bulkRemove(taskIds: string[]): Promise<void> {
+    try {
+      const savedObjectsToDelete = taskIds.map((taskId) => ({ id: taskId, type: 'task' }));
+      await this.savedObjectsRepository.bulkDelete(savedObjectsToDelete);
```

Comment:
It looks like if there's an error in the `savedObjectsRepository.delete` function, it would bubble up via the task manager delete back to the rules client delete, where it would throw. The `savedObjectsRepository.bulkDelete` function does something similar to the new bulk delete fn you where it returns an array of responses that might include an error. Therefore if you're bulk removing multiple tasks and one of them fails, right now it does not bubble back to the rules client. Wondering if it should?